backends/kv/writer_test.go

```diff
@@ -141,6 +141,20 @@ func (suite *WriterTestSuite) TestValidateFrameInputRepeatingShardingKey() {
 	suite.Equal("column 'col1' appears more than once as a partition key", err.Error())
 }
 
+func (suite *WriterTestSuite) TestValidateFrameInputInvalidColumnName() {
```

Comment:
Also add a positive test? One for name length?

Reply:
Added a test for name length
a positive already exist in that suite